pallets/pallet-bonded-coins/src/curves/mod.rs

```diff
@@ -87,6 +112,14 @@ where
 	}
 }
 
+/// Trait defining the bonding function for a curve.
+/// The bonding function is used to calculate the cost of purchasing or selling assets using the curve.
+/// The trait is implemented for each curve type.
+///
+/// Variables:
```

Comment:
These are called arguments.

Reply:
Updated in: [2ae4290](https://github.com/KILTprotocol/kilt-node/pull/801/commits/2ae429067b29b8adcd5200855c7b3e4dcfa0abe2)